src/client/mod.rs

```diff
@@ -396,7 +531,7 @@ pub struct TokenOptions {
     ttl: Option<String>,
     explicit_max_ttl: Option<String>,
     display_name: Option<String>,
-    num_uses: Option<u64>,
+    num_uses: Option<VaultNumUses>,
```

Comment:
Should this just be a `VaultNumUses` now, to illustrate Unlimited, or limited?

Reply:
Double checked documentation and so null shouldn't be possible. So I removed the Option.